eslint.config.js

```diff
@@ -17,7 +17,13 @@ module.exports = [
         }
     },
     {
-        files: ["eslint.config.js", "tests/**/*.js"],
+        files: ["eslint.config.js"],
+        rules: {
+            "n/no-unpublished-require": "off"
+        }
```

Comment:
We could also disable this rule globally, since there are no published files in this repo.

Reply:
you can add `"private": true` in `package.json` - the rule won't report in this case.

Reply:
Sorry, I totally forgot about this PR. If there's a better solution for this, feel free to submit a new one.